nservicebus/upgrades/5to6.md

```diff
@@ -680,3 +680,22 @@ snippet: 5to6-ExplicitProperties
 To access it at runtime create a new [`Feature`](/nservicebus/pipeline/features.md) and put the following code in the `.Setup` method 
 
 snippet: 5to6-IConfigureComponentsNotInjected
+
+# Configure type no longer available
+
+In Version 5 the configure type was used to provide runtime access to the local endpoint address, scanned types etc via dependency injection.
```

Comment:
perhaps end with some preamble? "these things can now be accesses with the following"?